crates/noir_field/src/generic_ark.rs

```diff
@@ -149,6 +149,10 @@ impl<F: PrimeField> FieldElement<F> {
         F::Params::MODULUS_BITS
     }
 
+    pub fn max_num_bytes() -> u32 {
+        Self::max_num_bits() / 8
```

Comment:
Is max_num_bits guaranteed to be divisible by 8? If not we should add 1 if necessary